blog/2020-12-09-eclipse-vert-x-4-released.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Eclipse Vert.x 4 released!
+category: releases
+authors:
+  - name: Julien Viet
+    github_id: vietj
+summary: >-
+  Vert.x 4 has just been released and it's the best thing since sliced buffers.
+---
+
+We are extremely pleased to announce the release of Vert.x 4.
+
+## What's new in Vert.x 4?
+
+Vert.x 4 is the evolution of the Vert.x 3.x series that brings [key features](https://vertx.io/blog/whats-new-in-vert-x-4/) to Vert.x.
+
+## From Vert.x 3 to Vert.x 4
+
+Vert.x 4 extends the Vert.x 3 line with a set of new features while providing the same Vert.x 3 experience:
+Vert.x 3 users will feel at home with Vert.x 4.
+
+Vert.x 3 users migrating to Vert.x 4 should read this [page](https://vertx.io/blog/from-vert-x-3-to-vert-x-4/).
+
+## Without the community we are nothing
+
+The [community](https://vertx-web-site.github.io/community/) of people made Vert.x 4 a reality.
+
+Special thanks go to Michel Krämer the creator of this absolutely stunning Website.
+
+## Finally
+
+The [deprecations and breaking changes](https://github.com/vert-x3/wiki/wiki/4.0.0-Deprecations-and-breaking-changes)
+ can be found on the wiki.
+
+For this release there are no Docker images (????)
```

Comment:
Maybe:

> For this release there are no Docker images. With Vert.x 4 we're going back to the true nature of being an unopinionated library. We focus on producing artifacts and you decide how to assemble them.

Reply:
I'm changing this we actually still have base docker images, but it would have been a good change.